gum/backend-dbghelp/gumsymbolutil-dbghelp.c

```diff
@@ -148,6 +148,28 @@ gum_find_functions_matching (const gchar * str)
   return matches;
 }
 
+guint64
```

Comment:
```suggestion
gboolean
```

Perhaps just a boolean to indicate success for now, as the base address may not be available in other backends.